### PR TITLE
Update references to proof-of-[word] to follow the style guidelines

### PR DESCRIPTION
**Pull request by vluna**
## Description
- Update references to Proof-of-[word] and proof-of-[word] across the pages following the [style guide](https://ethereum.org/en/contributing/style-guide/#proof-of).
- As per the comments in the issue, ignored titles of articles and resources for external links.

## Related Issue
[Proof-of-Stake not used consistently #4374](https://github.com/ethereum/ethereum-org-website/issues/4374)

### CODE REVIEW

File: src/intl/de/page-eth2-staking.json [hunk 1]
```diff
@@ -32,12 +32,12 @@
...
   "page-eth2-staking-pool-desc": "Wenn du weniger als 32 ETH hast, kannst du mit einem kleineren Einsatz zu einem Staking-Pool beitragen. Einige Unternehmen können das alles in deinem Namen tun, also musst du dir keine Sorgen machen online zu bleiben. Hier sind ein paar Unternehmen zum Informieren.",
-  "page-eth2-staking-pos-explained": "Proof-of-Stake erklärt",
-  "page-eth2-staking-pos-explained-desc": "Staking ist, was du tun musst, um ein Validator in einem Proof-of-Stake system zu werden. Dies ist ein Konsensmechanismus, der das derzeitige Proof-of-Work-System ersetzen wird. Konsensmechanismen sind das, was Blockchains wie Ethereum sicher und dezentralisiert machen.",
-  "page-eth2-staking-pos-explained-desc-1": "Proof-of-Stake trägt dazu bei, das Netzwerk auf verschiedene Weise zu sichern:",
+  "page-eth2-staking-pos-explained": "Proof-of-stake erklärt",
+  "page-eth2-staking-pos-explained-desc": "Staking ist, was du tun musst, um ein Validator in einem proof-of-stake system zu werden. Dies ist ein Konsensmechanismus, der das derzeitige proof-of-work-wystem ersetzen wird. Konsensmechanismen sind das, was Blockchains wie Ethereum sicher und dezentralisiert machen.",
```
Comment (minimalsm) [inline src/intl/de/page-eth2-staking.json:36]:
```suggestion
  "page-eth2-staking-pos-explained-desc": "Staking ist, was du tun musst, um ein Validator in einem proof-of-stake system zu werden. Dies ist ein Konsensmechanismus, der das derzeitige proof-of-work-system ersetzen wird. Konsensmechanismen sind das, was Blockchains wie Ethereum sicher und dezentralisiert machen.",
```

File: src/intl/de/page-eth2-staking.json [hunk 2]
```diff
@@ -49,12 +49,12 @@
   "page-eth2-staking-title-2": "Risiken",
   "page-eth2-staking-title-3": "Anforderungen",
   "page-eth2-staking-title-4": "Wie man sein ETH einsetzt",
-  "page-eth2-staking-upgrades-li": "Proof-of-Stake wird durch die Beacon Chain verwaltet.",
-  "page-eth2-staking-upgrades-li-2": "Ethereum wird in der absehbaren Zukunft eine Proof-of-Stake-Beacon Chain und ein Proof-of-Work-Mainnet haben. Mainnet ist das Ethereum, das wir schon seit Jahren benutzen.",
+  "page-eth2-staking-upgrades-li": "Proof-of-stake wird durch die Beacon Chain verwaltet.",
+  "page-eth2-staking-upgrades-li-2": "Ethereum wird in der absehbaren Zukunft eine proof-of-stake-beacon Chain und ein proof-of-work-mainnet haben. Mainnet ist das Ethereum, das wir schon seit Jahren benutzen.",
```
Comment (minimalsm) [inline src/intl/de/page-eth2-staking.json:53]:
```suggestion
  "page-eth2-staking-upgrades-li-2": "Ethereum wird in der absehbaren Zukunft eine proof-of-stake-beacon Chain und ein proof-of-work-Mainnet haben. Mainnet ist das Ethereum, das wir schon seit Jahren benutzen.",
```

File: src/intl/pl/page-eth2-staking.json [hunk 3]
```diff
@@ -32,7 +32,7 @@
   "page-eth2-staking-more-sharding": "Więcej o shardingu",
   "page-eth2-staking-pool": "Stakowanie w puli",
   "page-eth2-staking-pool-desc": "Jeśli masz mniej niż 32 ETH, będziesz mógł dodać mniejszą stawkę do puli stawek. Niektóre firmy mogą to wszystko zrobić w Twoim imieniu, więc nie musisz się martwić o pozostanie online. Oto kilka firm do sprawdzenia.",
-  "page-eth2-staking-pos-explained": "Wyjaśnienie Proof-of stake",
+  "page-eth2-staking-pos-explained": "Wyjaśnienie proof-of stake",
```
Comment (minimalsm) [inline src/intl/pl/page-eth2-staking.json:35]:
```suggestion
  "page-eth2-staking-pos-explained": "Wyjaśnienie proof-of-stake",
```

File: src/content/translations/de/eth2/merge/index.md [hunk 4]
```diff
@@ -20,15 +20,15 @@ summaryPoints:
...
 
 Stelle dir Ethereum als Raumschiff vor, das noch nicht ganz für seine interstellare Reise bereit ist. Mit der Beacon Chain und Shard Chains hat die Community einen neuen Motor und einen gestählten Rumpf konstruiert. Wenn es an der Zeit ist, werden diese neuen Systeme an das Raumschiff angedockt, werden ein Teil davon; bereit, um Lichtjahre in die Zukunft zu springen.
 
 ## Andocken ans Mainnet {#docking-mainnet}
 
-Wenn alles bereit ist, wird das Ethereum-Mainnet an die Beacon Chain "angedockt" und zu seinem eigenen Shard werden, die statt [Proof-of-Work](/developers/docs/consensus-mechanisms/pow/) das Proof-of Stake-Verfahren anwendet.
+Wenn alles bereit ist, wird das Ethereum-Mainnet an die Beacon Chain "angedockt" und zu seinem eigenen Shard werden, die statt [proof-of-work](/developers/docs/consensus-mechanisms/pow/) das proof-of Stake-Verfahren anwendet.
```
Comment (minimalsm) [inline src/content/translations/de/eth2/merge/index.md:29]:
```suggestion
Wenn alles bereit ist, wird das Ethereum-Mainnet an die Beacon Chain "angedockt" und zu seinem eigenen Shard werden, die statt [proof-of-work](/developers/docs/consensus-mechanisms/pow/) das proof-of-stake-Verfahren anwendet.
```

**Review by minimalsm — CHANGES_REQUESTED** [review]

**Review by minimalsm — APPROVED** [review]
Wow thanks for this @vluna 🥰😍

I knew there were a few but didn't realise there were all these inconsistencies. LGTM just waiting to double check the build before bringing this in.

**Comment by vluna** [comment]
> Wow thanks for this @vluna 🥰😍
> 
> I knew there were a few but didn't realise there were all these inconsistencies. LGTM just waiting to double check the build before bringing this in.

No worries @minimalsm, happy to help 😄

I have updated the references based on your comments and also a few more variations just to confirm. Let me know if anything else needs to be changed.

**Comment by minimalsm** [comment]
Merging now 😀❤️

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳!

**Pull request merged**